ban/http/schema.py

```diff
@@ -24,6 +24,15 @@
         'url': 'https://adresse.data.gouv.fr/api-gestion/',
     },
     'paths': {},
+    'tags': [
+        {'name': 'Municipality'},
+        {'name': 'Postcode'},
+        {'name': 'Group'},
+        {'name': 'Housenumber'},
+        {'name': 'Position'},
+        {'name': 'Diff'},
+        {'name': 'User'},
+    ],
```

Comment:
Right, because we want NOW to group each routes by model.
But, if we want to define other groups like 'technic' where we put 'Diff', 'User', ...?